pkg/transform/snowplow_gtmss_preview_test.go

```diff
@@ -30,6 +34,7 @@ func TestGTMSSPreview(t *testing.T) {
 			Ctx:       "contexts_com_google_tag-manager_server-side_preview_mode_1",
 			Property:  "x-gtm-server-preview",
 			HeaderKey: "x-gtm-server-preview",
+			Expiry:    fiftyYears,
```

Comment:
2069 devs be like "why my test failing?" :D

Reply:
in 2069 I'm an actual senior engineer and I can help!